sorted-list mc mirror failed with 60gigs copy

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x4343dd]

goroutine 24 [running]:
main.(*sortedList).Create(0xc820509f08, 0x7efdf0295b08, 0xc8201198f0, 0xc820119d10, 0xc, 0xc820119d10)
	/home/harsha/mygo/src/github.com/minio/mc/sorted-list.go:94 +0xb1d
main.deltaSourceTargets.func1(0xc820070480, 0xc820119900, 0x1, 0x1, 0x7efdf0295b08, 0xc8201198f0)
	/home/harsha/mygo/src/github.com/minio/mc/mirror-url.go:112 +0x283
created by main.deltaSourceTargets
	/home/harsha/mygo/src/github.com/minio/mc/mirror-url.go:170 +0x89

goroutine 1 [runnable]:
main.doPrepareMirrorURLs(0xc8200f3c20, 0xc820114540)
	/home/harsha/mygo/src/github.com/minio/mc/mirror-main.go:168 +0x782
main.doMirrorCmdSession(0xc8200f3c20)
	/home/harsha/mygo/src/github.com/minio/mc/mirror-main.go:215 +0x154
main.mainMirror(0xc8200f8780)
	/home/harsha/mygo/src/github.com/minio/mc/mirror-main.go:336 +0x561
github.com/minio/mc/vendor/github.com/minio/cli.Command.Run(0x948608, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0x9e2c80, 0x45, 0x0, ...)
	/home/harsha/mygo/src/github.com/minio/mc/vendor/github.com/minio/cli/command.go:123 +0x10be
github.com/minio/mc/vendor/github.com/minio/cli.(*App).Run(0xc8200be300, 0xc82006e100, 0x4, 0x4, 0x0, 0x0)
	/home/harsha/mygo/src/github.com/minio/mc/vendor/github.com/minio/cli/app.go:200 +0x77d
github.com/minio/mc/vendor/github.com/minio/cli.(*App).RunAndExitOnError(0xc8200be300)
	/home/harsha/mygo/src/github.com/minio/mc/vendor/github.com/minio/cli/app.go:211 +0x45
main.main()
	/home/harsha/mygo/src/github.com/minio/mc/main.go:195 +0x65

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1696 +0x1

goroutine 20 [syscall]:
```

--- a/sorted-list.go
+++ b/sorted-list.go
@@ -90,8 +90,10 @@ func (sl *sortedList) Create(clnt client.Client, id string) *probe.Error {
 					if content.Content.Type.IsDir() && (content.Content.Type&os.ModeSymlink == os.ModeSymlink) {
 						continue
 					}
+					errorIf(content.Err.Trace(), fmt.Sprintf("Skipping ‘%s’.", content.Content.Name))
+					continue
 				}
-				errorIf(content.Err.Trace(), fmt.Sprintf("Skipping ‘%s’.", content.Content.Name))
+				errorIf(content.Err.Trace(), "Skipping unknown file.")
 				continue
 			}
 			return content.Err.Trace()

